br/pkg/restore/split.go

```diff
@@ -96,8 +96,8 @@ func (rs *RegionSplitter) Split(
 	if errSplit != nil {
 		return errors.Trace(errSplit)
 	}
-	minKey := codec.EncodeBytes(sortedRanges[0].StartKey)
-	maxKey := codec.EncodeBytes(sortedRanges[len(sortedRanges)-1].EndKey)
+	minKey := codec.EncodeBytes(sortedRanges[0].StartKey, nil)
```

Comment:
Should it be `codec.EncodeBytes(nil, sortedRanges[0].StartKey)`?

Reply:
you are right!